expression/builtin_time.go

```diff
@@ -449,12 +449,10 @@ func (c *timeDiffFunctionClass) getFunction(ctx sessionctx.Context, args []Expre
 	if err != nil {
 		return nil, err
 	}
-
-	bf.tp.Decimal = fsp
+	bf.setDecimalAndFlenForTime(fsp)
 
 	var sig builtinFunc
 	// arg0 and arg1 must be the same time type(compatible), or timediff will return NULL.
-	// TODO: we don't really need Duration type, actually in MySQL, it use Time class to represent
```

Comment:
Could you please polish these comments?

Reply:
I don't think this TODO makes sense, so I remove it.